var/spack/repos/builtin/packages/libbson/package.py

```diff
@@ -30,14 +30,12 @@ class Libbson(AutotoolsPackage):
     parsing, and iterating BSON documents."""
 
     homepage = "https://github.com/mongodb/libbson"
-    url      = "https://github.com/mongodb/libbson/releases/download/1.6.1/libbson-1.6.1.tar.gz"
+    url      = "https://github.com/mongodb/libbson/releases/download/1.6.2/libbson-1.6.2.tar.gz"
 
+    version('1.6.2', 'c128a2ae3e35295e1176465be60f19db')
     version('1.6.1', '4d6779451bc5764a7d4982c01e7bd8c2')
 
     depends_on('autoconf', type='build')
     depends_on('automake', type='build')
     depends_on('libtool', type='build')
     depends_on('m4', type='build')
-
-    # 1.6.1 tarball is broken
-    force_autoreconf = True
```

Comment:
I'm not sure what we want to do here. As is, your PR makes 1.6.1 no longer work. Normally I would just remove 1.6.1, but some people might depend on it: https://groups.google.com/forum/#!topic/spack/afI3-EbiglM

If we want to keep 1.6.1, we should probably also keep `force_autoreconf = True`. If we don't, then we can get rid of all of the dependencies as well because they are only needed to regenerate the configure script.